tests/components/test_rflink.py

```diff
@@ -4,21 +4,22 @@
 
 from homeassistant.bootstrap import async_setup_component
 from homeassistant.components.rflink import (
-    CONF_RECONNECT_INTERVAL, SERVICE_SEND_COMMAND)
+    CONF_RECONNECT_INTERVAL, SERVICE_SEND_COMMAND, RflinkCommand)
 from homeassistant.const import (
     ATTR_ENTITY_ID, SERVICE_TURN_OFF, SERVICE_STOP_COVER)
 
 
-async def mock_rflink(hass, config, domain, monkeypatch, failures=None):
+async def mock_rflink(hass, config, domain, monkeypatch, failures=None,
+                      failcommand=False):
     """Create mock RFLink asyncio protocol, test component setup."""
     transport, protocol = (Mock(), Mock())
 
     async def send_command_ack(*command):
-        return True
+        return not failcommand
     protocol.send_command_ack = Mock(wraps=send_command_ack)
 
     def send_command(*command):
-        return True
+        return not failcommand
     protocol.send_command = Mock(wraps=send_command)
```

Comment:
Added `failcommand`parameter for testing purposes. Simulates `send_command` and `send_command_ack` failures.